apps/omg_watcher/lib/omg_watcher/web/endpoint.ex

```diff
@@ -18,6 +18,11 @@ defmodule OMG.Watcher.Web.Endpoint do
 
   socket("/socket", OMG.Watcher.Web.Socket, websocket: [])
 
+  # It's important that this is at the top of `endpoint.ex`, before any other plugs.
+  if Application.get_env(:omg_watcher, :sql_sandbox) do
+    plug(Phoenix.Ecto.SQL.Sandbox)
```

Comment:
From my best understanding we don't use ecto through Phoenix, so it's enough to plug the sandbox to the repo. Phoenix controllers are unaware of ecto existence. Please revert.